### PR TITLE
Add James Duong to committers

### PR DESCRIPTION
**Pull request by jduo**
Add myself to the committers list, per https://lists.apache.org/thread/yk01t5j0p26vjfr4vzxv1zls5zk2hk4v

### CODE REVIEW

**Review by kou — APPROVED** [review]
+1

Welcome!

File: _data/committers.yml [hunk 1]
```diff
@@ -296,6 +296,10 @@
   role: Committer
   alias: assignuser
   affiliation: Voltron Data
+- name: James Duong
+  role: Committer
+  alias: jduo
```
Comment (kou) [inline _data/committers.yml:301]: Is your Apache ID is `judong` not `jduo`?

https://people.apache.org/phonebook.html?uid=jduong

```suggestion
  alias: jduong
```

Comment (jduo) [reply _data/committers.yml:301]: Oh that's correct. My apache is is jduong and my GitHub user is jduo. Thanks

**Comment by kou** [comment]
@jduo Do you want to merge this by yourself as a new committer? :-)

**Comment by jduo** [comment]
Thanks @kou !

**Pull request merged**